.github/workflows/docker.yml

```diff
@@ -1,8 +1,10 @@
 name: docker-push-pr
 
 on:
+  push:
+    branches:
+      - develop
   pull_request:
```

Comment:
Can `pull_request` just be a naked key without any value?

Reply:
Yep it matches to all pull requests